igor.dzetcyna/1/lesson_2/task1-class_Pet.rb

```diff
@@ -0,0 +1,112 @@
+class Pet
+  attr_accessor :name, :health, :eat, :play, :sleep
+ 
+   def initialize (name, health, eat, play, sleep)
```

Comment:
method `initialize` used to set some params it's pretty bad idea to write whole your logic here, please, separate it to methods